### ISSUE TITLE
Explain purpose of promise in signupActions.js?

### ISSUE THREAD

**Issue by norbertschuler** [Plant-for-the-Planet-org/treecounter-app#1544]
There is an unhandled rejection of a promise as following in #1008:

<img width="612" alt="Bildschirmfoto 2019-10-13 um 11 34 35" src="https://user-images.githubusercontent.com/1532418/66713644-78017f80-edad-11e9-988e-627b53466192.png">

I wonder if it's related to this promise which I do not exactly understood the purpose of:

https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/036c155d790d6a59749ec934dbd8e70db223360c/app/actions/signupActions.js#L60

**Comment by mahmed0715** [comment]
There might be use case where the coder wanted promise to be returned, but one thing to confirm that this action must be called in async/await or .then and must be catched if error. if this is confirmed then I personally dont see issue with it. @norbertschuler 
@rabiul042 can you please check this.

**Comment by norbertschuler** [comment]
I just tested that this promise does not cause any error and simple seems to have been used as the `function signUp` needs to return a promise. The unhandled rejection must have had another reason. Some time ago I added `.catch()` blocks to promises within our source code where they were missing. Maybe this bug is already fixed. As it has not happened within the last month I wouldn't put any more effort in trying to find it.
